README.md

```diff
@@ -257,3 +299,20 @@ Destination is relative to the output policy set (`out/masterfiles`).
 - Enable `services_autorun_bundles` class in `def.json`.
 - Merge any `def.json` recursively into `out/masterfiles/def.json`.
 - Copy any other files with their existing directory structure to destination.
+
+#### `bundles <bundles ...>`
+- Ensure bundles are evaluated by adding them to the bundle sequence, using `def.json`.
+- Only manipulates the bundle sequence, to ensure policy files are copied and parsed, use other build steps, for example `copy` and `policy_files`.
```

Comment:
Specifically, this adds to `default:def.control_common_bundlesequence_end` right? That's an MPFism, so I think it makes sense to call it out specifically.

Reply:
I can try to make this a bit more clear, but in general MPF will be assumed / required for some features of CFEngine and cfbs.

Reply:
Fixed.